app/views/calculators/guaranteed_income/_estimate.html.erb

```diff
@@ -0,0 +1,19 @@
+<p>
+  Based on what you’ve told us, if you take your <%= number_to_currency(pot) %> pension pot as an annuity, you could get:
```

Comment:
The 2nd one, so:

'Based on what you’ve told us, if you use your <%= number_to_currency(pot) %> pension pot to buy an annuity, you could get:'